### ISSUE TITLE
Cannot use keyboard to control flameshot and cannot open flameshot from click the icon

### ISSUE THREAD

**Issue by m3ng9i** [flameshot-org/flameshot#501]
**Flameshot version**
```
$ flameshot --version
Flameshot 0.6.0-11(Debian)
Compiled with Qt 5.11.3
```

**Describe the bug**

1. Cannot use keyboard to control flameshot
2. Cannot type text into the screenshot
3. Cannot open flameshot from click the icon

**To Reproduce**

Problem 1: When openning flameshot use command 'flameshot gui', the flameshot interface shows, but I cannot use any keyboard buttons to control flameshot (like Esc, Enter, Ctrl + C), nothing happens. The mouse click works.

Problem 2: Cannot type any text into the screenshot image, nothing happens. If I take a screenshot of terminal (or text editor) and try to input something to the screenshot (nothing happens) and then if I close flameshot, the text will show on terminal (or text editor).

Problem 3: Click flameshot's icon from Gnome3's application menu, nothing happens. Run command 'flameshot' in terminal, nothing happens except an error:

`Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.`

If I run `QT_QPA_PLATFORM=wayland flameshot`, there's another error:

```
    Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
    qt.qpa.plugin: Could not find the Qt platform plugin "wayland" in ""
    This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

    Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, xcb.
```

The only way to let flameshot shows is run `flameshot gui` or `flameshot config`.

**Expected behavior**

1. Use keyboard button to control flameshot.
2. Type text into the screenshot image.
3. Open flameshot from click the icon.

**System Information**
```
$ gnome-shell --version
GNOME Shell 3.30.2

OS: Debian testing buster
Kernel: x86_64 Linux 4.19.0-4-amd64

I installed flameshot with apt command:

    sudo apt install flameshot

```

**Comment by m3ng9i** [comment]
After I switch to Xorg from Wayland, all the problems described above are gone.

**Comment by atten** [comment]
The same thing on Ubuntu 18.04, KDE 5.47.0

**Comment by bwoodruff** [comment]
<details><summary>$ QT_QPA_PLATFORM=wayland flameshot </summary>

```
QSocketNotifier: Can only be used with threads started with QThread
QObject::connect: No such signal QPlatformNativeInterface::systemTrayWindowChanged(QScreen*)
```
</details>

It seems to work properly (or at least better) if instead I run

`$ QT_QPA_PLATFORM=xcb flameshot`

**Comment by borgmanJeremy** [comment]
I think this is resolved now per #1282
